### ISSUE TITLE
I have some problems

### ISSUE THREAD

**Issue by paulohsrocha** [WHSuite/whsuite#5]
I have some problems with Whsuite. I need urgent help.

1. I can not find any information on how to use Whsuite on my local server (WAMP on Windows 10). It worked on my online server, but I would also install on my local server for testing.

2. The system did not come with any addon and can not find them for download, such as cPanel, Helpdesk, etc. The 01_Installing_Addons page documentation is blank and found nothing about the Internet.

3. I can not register any product because it has no Addon.

4. How do I translate Whsuite to Portuguese? I am willing to do the translation and contribute.

**Comment by ngalfas** [comment]
Don't worry, the same goes here on a fresh install.  Although this project looks very promising, i think it needs a team behind it for further support and development.

As you said, there is no way i managed to make the addons work. I uploaded the .zip files of addons to the folder (which i had to create) but the addons screen shows nothing. 
At first i tried uploding the uncompressed folder of each addon, still nothing.
As you said there is no documentation about installing addons.

As i see it, we should wait for a community to be grown to deal with these problems.
As you said, a fresh install can't work properly as it is.

**Comment by paulohsrocha** [comment]
Exactly.

I thought there was a team of committed developers and even contribute financially. But I do not find any information, not even to begin.

**Comment by desfire** [comment]
It used to work on their beta version, it makes no sense it's not working now..

**Comment by DACRepair** [comment]
Since they open sourced this, it has some issues (they had to remove things that were not open, to comply with licensing). As far as how one would do the questioned things Paulo, here are what I can tell you from what I can dig through:
1. Make sure you have all the needed PHP plugins available, easiest way to tell is use phpinfo(), and seeing if all plugins are loaded (I have added an attachment including a test.php file you can drop in your htdocs, and view in a browser to see what is actually going on).
2. check the admin panel under Settings, Addon Management. If you don't see anything in there, check that you have all the PHP plugins are working.
3. This is true, make sure that you were able to get 2 to work.
4. This I have not looked in to, but if you look at WHSuite's account, there are a bunch of additional resources ( [https://github.com/WHSuite](https://github.com/WHSuite) ).

test.php file (zipped)
[test.zip](https://github.com/WHSuite/whsuite/files/560608/test.zip)

**Comment by bigalgeorge** [comment]
Hi guys. I went through the php code for addons and where they are failing is in naming conventions.
If you unzip these and put them in your whsuite/app/addons directory they will work and you will understand how to name them eg paypal/paypal_details.php gives naming convention of directory. You can edit other addons the same. Hope this helps.
[addons.zip](https://github.com/WHSuite/whsuite/files/573063/addons.zip)

**Comment by ngalfas** [comment]
George you nailed it. I confirm that your zip addons-structure folder is correct and is the way to go for newcomers since we have no idea of the corect file structure from previous working installs.

So, anyone who tries a fresh install should manually create the whsuite/app/addons folder
and unzip the (https://github.com/WHSuite/whsuite/files/573063/addons.zip) file inside
Then you will have your first basic addons showup for installation !

I didn't have any dependencies problem, i just uploaded the uncompressed folders and it worked.

**Comment by bigalgeorge** [comment]
Appreciate. I did check out the recommended _A much easier way of brining them in would be to add the addons to your main whsuite composer.json file_ but have no idea what he means.
Next step is to get some inspired developers to take on board this project as it has greatest potential Opensource. Im capable of PHP but not so hot on OOp and DB methods.
I will help with documentation and some PHP if anyone likes to contribute please do so, and original developers please PM me when you have time; youve done too much to let it die a natural.

**Comment by rickmills** [comment]
@bigalgeorge Yep you've got it right with the addons. Basically the git repos being called the same as the naming convention we used was an issue. 

We previously had WHSuite set up so that when we did a build for a release, our main composer.json file contained all the addons. We then did a composer install and brought them in. Using composer we were able to specify that any addon packages were placed into the correct location.

This was before we ran everything on github however (we used BitBucket and a Robo task runner during development), so the naming of the git repos wasn't a problem. 

I'll see about getting those docs sorted out :)

**Comment by paulohsrocha** [comment]
Well... A billing system automation for me need to be easy to use, or is not good. 

I go back when there is a system working.

:(
